app/overview/page.tsx

```diff
@@ -0,0 +1,136 @@
+'use client'
+
+import React from "react";
+import ToggleButton from "../components/toggleButton";
+import Image from "next/image";
+import arrow from "../../public/arrow.svg";
+import eth from "../../public/coin-logos/eth-logo.png"
+import usdc from "../../public/coin-logos/usdc-logo.png"
+import { Open_Sans } from "next/font/google";
+import AdditionalButton from "../components/additional-button";
+
+const openSans = Open_Sans({ subsets: ["latin"] });
+export default function page() {
+
+  
+  const tokenPairs = [
+    { id: 1, from: { name: 'Ethereum', symbol: 'ETH', logo: eth }, to: { name: 'USDC', symbol: 'USDC', logo: usdc }, percentage: '50', enabled: true },
+    { id: 2, from: { name: 'Ethereum', symbol: 'ETH', logo: eth }, to: { name: 'USDC', symbol: 'USDC', logo: usdc }, percentage: '50', enabled: true },
+    { id: 3, from: { name: 'Ethereum', symbol: 'ETH', logo: eth }, to: { name: 'USDC', symbol: 'USDC', logo: usdc }, percentage: '50', enabled: true },
+    { id: 4, from: { name: 'Ethereum', symbol: 'ETH', logo: eth }, to: { name: 'USDC', symbol: 'USDC', logo: usdc }, percentage: '50', enabled: false },
+    { id: 5, from: { name: 'Ethereum', symbol: 'ETH', logo: eth }, to: { name: 'USDC', symbol: 'USDC', logo: usdc }, percentage: '50', enabled: true },
+  ];
+
+  const handlePercentageChange = (e: React.ChangeEvent<HTMLSelectElement>, id: number) => {
+    console.log(e, id);
+  };
+
```

Comment:
Hi @melnikga could you ensure the state is being updated anytime the user clicks on the percentage select field.

Reply:
Hi @Abeeujah I'll fix it within a couple of hours

Reply:
@Abeeujah I fixed that